lib/solidus_subscriptions/engine.rb

```diff
@@ -11,9 +12,17 @@ class Engine < Rails::Engine
     end
 
     config.autoload_paths << config.root.join('app', 'jobs')
-    config.autoload_paths << config.root.join('lib')
 
-    initializer 'solidus_subscriptions.configs', before: "spree.register.payment_methods" do
+    initializer 'require subscription lib helpers' do
+      require 'solidus_subscriptions/permitted_attributes'
```

Comment:
Good catch! I didnt mean to add it in twice. You might be right. I wasnt sure if it was better to require this in an initializer or simple at the top of the engine along side spree/core.

I think I prefer the latter, so I will make that change :)